code/game/objects/items/food/sandwichtoast.dm

```diff
@@ -284,7 +284,7 @@
 	// Closest thing to a mullet we have
 	if(consumer.hairstyle == "Gelled Back" && istype(consumer.get_item_by_slot(ITEM_SLOT_ICLOTHING), /obj/item/clothing/under/rank/civilian/cookjorts))
 		return FOOD_LIKED
-	return FOOD_DISLIKED
+	return FOOD_ALLERGIC
```

Comment:
You already get a specific disease when you eat it without meeting the requirements, so why?

Reply:
Quote from PR 

> Minor implementation detail: I added a "dangerous" level of food quality / scale, so I gave it to the Death Sandwich.

Food allergic is just synonymous with "this food will straight up kill you", it doesn't actually give you anaphylaxis unless you actually have an allergy. 

If you'd like me to rename it though